modules/oadp-operator-supported.adoc

```diff
@@ -11,7 +11,7 @@
 |===
 
 |Version
-|{OCP-short} version
+|OCP version
```

Comment:
```suggestion
|{product-title} version
```

Reply:
This is incorrect. this will not render in a table in the OCP enterprise